HACKING

```diff
@@ -262,40 +251,25 @@ project's source code.
 
 What happens to all the branches?
 
-Leave the branches alone, approved branches are cleared automatically by
-Launchpad.
+Leave the branches alone, approved branches will be removed after they
+have been merged.
 
-For larger feature branches, use the team in Launchpad to allow other
+For larger feature branches, you can add collaborators on Github to allow other
 developers to work on the code with you.
 
 What if I want to help out on an existing merge request that I can't push to?
 
-    bzr branch lp:~OTHERPERSON/gala/fix-123456
-    cd fix-123456
-    # make commits
-    bzr push lp:~USERNAME/gala/fix-123456
-    bzr lp-open
-
-And in the Launchpad web overview of your branch, propose your branch for
-merging into lp:~OTHERPERSON/gala/fix-123456
+Similarly to with master you can create pull requests to other branches on Github,
+if the pull request is from a fork you will have to clone that fork first however.
 
 Updating a branch that may be out of sync with trunk:
 
-    bzr pull
-    bzr: ERROR: These branches have diverged
-    bzr merge lp:gala
+    git pull
+    git: Automatic merge failed; fix conflicts and then commit the result.
     # Hand-edit conflicting changes
-    bzr resolve FILENAME
+    git add FILENAME
     # If any conflicts remain continue fixing
-    bzr commit -m 'Merge changes from lp:gala'
-
-Save a little bandwidth, branch from an existing local copy that you keep
-around:
-
-    bzr branch lp:gala gala
-    bzr branch gala/ gala-fix-123456
-    cd gala-fix-123456
-    bzr pull lp:gala
+    git commit -m 'Merge master into fix-123456'
 
```

Comment:
I would encourage the use of "git fetch; git rebase origin/master" instead of merge.

Reply:
Generally pull requests are squashed which does this anyway, and since it is likely that the pull request will take a while to be reviewed (at least if it is significant) master will end up being merged into it which will undermine the rebase.